build.sbt

```diff
@@ -21,7 +24,14 @@ lazy val tutDirectoriesSettings = Seq(
   tutTargetDirectory := sourceDirectory.value / "jekyll"
 )
 
-lazy val docsSettings = buildSettings ++ dependencies ++ tutSettings ++ tutDirectoriesSettings
+lazy val noPublishSettings = Seq(
```

Comment:
I'd followed the instructions at https://github.com/47deg/microsites/blob/master/jekyll/docs.md#step-1-create-a-new-sbt-project-called-docs - perhaps I don't need it, I'll check and remove it.